gatling-core/src/main/resources/gatling-defaults.conf

```diff
@@ -43,6 +43,7 @@ gatling {
   charting {
     noReports = false       # When set to true, don't generate HTML reports
     maxPlotPerSeries = 1000 # Number of points per graph in Gatling reports
+    useGroupDurationInsteadOfCumulatedResponseTime = false  # Switch group timings from cumulated response time to group duration.
```

Comment:
Being able to handle both duration and cumulated response time together would require a lot of work and consideration:
* how to deal with the stats table on the Global page?
* what about assertions on groups, the API only let you target one metric?

I'm completely fine with the implementation you suggest. My only issue is with the option name, which meaningful, but very long. What about `useGroupDurationMetric`? The explaining comment looks sufficient to me so users know what they're doing.

Then, there's one thing missing from your PR: documentation. Could you please add some documentation, probably in `reports.rst`?

Thanks!